docs/pywbemclicmdshelp.rst

```diff
@@ -686,23 +686,26 @@ The following defines the help output for the `pywbemcli connection add --help`
 
 ::
 
-    Usage: pywbemcli connection add [COMMAND-OPTIONS]
+    Usage: pywbemcli connection add [COMMAND-OPTIONS] NAME
 
       Add a new WBEM connection definition from specified options.
 
       Create a new WBEM connection definition in the connections file from the
       specified options. A connection definition with that name must not yet
       exist.
 
-      The current connection remains unchanged by this command.
+      The NAME argument MUST exist. It define the server uri and the unique name
```

Comment:
Multiple issues in: "It define the server uri and the unique name under which this server connection information will be stored.":
* Grammar: "it define"
* It does not define the server URI.
If we follow my suggestion on changing the sentence on line 693, the whole sentence can be removed.